schemas/skills/samples/complex-skillmanifest-2.1.preview-0.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/skills/skill-manifest-2.1.preview-0.json",
+    "$schema": "https://schemas.botframework.com/schemas/skills/skill-manifest-2.1.preview-0.json",
     "$id": "SkillBot",
     "name": "Sample skill definition that can handle multiple types of activities",
```

Comment:
One thing the Power-VA team raised was the need to localise fields like name/description - there's a balance in my view between a manifest describing the service for purposes of communicating and something like a store/directory registration